corehq/apps/users/forms.py

```diff
@@ -1332,6 +1346,15 @@ class UserFilterForm(forms.Form):
         label=ugettext_noop("Location"),
         required=False,
     )
+    selected_location_only = forms.BooleanField(
+        required=False,
+        label=_('Only include mobile workers at the selected location'),
+    )
+    user_active_status = forms.ChoiceField(
```

Comment:
Given that there are only 3 choices, this would be a good place to save users a click with a SelectToggle widget, like the `columns` input does.

Reply:
Could be, but I tried to stay consistent with the [spec doc](https://docs.google.com/document/d/1354T85ioFXeLFpZCBBYlfJNzMPgnjMOUzgcJwlmK7hA/edit?pli=1#). If you think this UI needs to be reevaluated I can always take this up with Cody.

Reply:
Yeah, could you mention it to Cody? I don't think it'll be a controversial change. There's background on the relevant HQ standards [here](https://www.commcarehq.org/styleguide/molecules/#molecules-selections).

The spec looks like a select2 widget, btw, also described in those docs. If you decide to keep this as a a dropdown, you should be able to just add the `hqwebapp-select2` class to the crispy form and add `hqwebapp/js/widgets` as a dependency in your js.

Reply:
I'll take it up then :+1:

Reply:
I have updated the UI to have the SelecToggle behaviour.

Reply:
nice, thank you